src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -72,7 +72,7 @@ async fn upload_deduped(report: &CrashReport, container: &BlobContainerUrl) -> R
         .header("If-None-Match", "*")
```

Comment:
I should've done this originally, but in the comment above this, can we add a link to these docs: https://docs.microsoft.com/en-us/rest/api/storageservices/specifying-conditional-headers-for-blob-service-operations